Multisig/Data/Services/Safe Client Gateway Service/ProposeTransactionRequest.swift

```diff
@@ -67,13 +68,17 @@ extension SafeClientGatewayService {
         return asyncExecute(request: ProposeTransactionRequest(safe: transaction.safe!,
                                                                sender: sender,
                                                                signature: signature,
-                                                               transaction: transaction),
+                                                               transaction: transaction,
+                                                               chainId: chainId),
                             completion: completion)
     }
 
     func proposeTransaction(transaction: Transaction, sender: AddressString, signature: String) throws {
```

Comment:
e.g. here transaction should already have attached safe, so chainId could be fetched as `transaction.safe.network!.id`